.github/workflows/archive-extension-source-code.yml

```diff
@@ -0,0 +1,67 @@
+name: Archive Extension Build And Source Code
+
+on:
+  push:
+    tags:
+      - coil-extension@*
+
+jobs:
+  save-mono-repo-source-code:
+    runs-on: ubuntu-latest
+   
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Step 1 - Use the Upload Artifact GitHub Action to upload the mono repo
+        uses: actions/upload-artifact@v2
+        with: 
+          name: source-${{env.RELEASE_VERSION}}
+          path: ${{ github.workspace }}
+
+  save-chrome-build:
+    runs-on: ubuntu-latest
+    needs: [ save-mono-repo-source-code ]
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
you don't really need the checkout right?

Reply:
I take it you do need it for something ?

Reply:
yes, we do need the checkout if we want to perform any operations with the content of the repository

Reply:
resolved by replacing git checkout with "checking out" the archive